api/v1/runtimecomponent_types.go

```diff
@@ -195,6 +195,10 @@ type RuntimeComponentServiceAccount struct {
 	// Name of the service account to use for deploying the application. A service account is automatically created if it's not specified.
 	// +operator-sdk:csv:customresourcedefinitions:order=2,type=spec,displayName="Service Account Name",xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	Name *string `json:"name,omitempty"`
+
+	// Don't verify that the service account has a valid pull secret. Defaults to false
```

Comment:
Suggestion: `Skip verifying that the service account has a valid pull secret. Defaults to false.`